### DISCUSSION TITLE
Context "themeManager" not found, do you forget to inject it?

### DISCUSSION THREAD

**Discussion by sytolk** [Milkdown/milkdown#813] [General]
Hi,
I have use Switch Theme like this:
https://github.com/Saul-Mirone/milkdown/blob/824e89682e183f6af2179fb09de43802cb1dffb3/website/pages/theme/using-themes/index.md#switch-theme

but after update milkdown to v6.5.2 this error appears: `Context "themeManager" not found, do you forget to inject it?`
![FC885DC4-EC03-4E53-BAF0-A6EA1B8D1D10](https://user-images.githubusercontent.com/1415463/201305932-92f8ca14-98f8-4f56-b5a9-d324892854c7.png)

**Comment by Saul-Mirone** [comment]
I can't reproduce the case, maybe reinstall dependencies can help?
https://codesandbox.io/s/clever-pare-uvuovz?file=/src/index.ts

**Comment by sytolk** [comment]
I have done with React and this works.. https://codesandbox.io/s/eager-platform-60tj12?file=/src/App.js
maybe in useEffect theme switch have problem
